app/src/main/java/com/infomaniak/mail/utils/MessageBodyUtils.kt

```diff
@@ -39,13 +40,13 @@ object MessageBodyUtils {
         "#origbody",
         "#oriMsgHtmlSeperator",
         "#reply139content",
-        ".gmail_extra",
-        ".gmail_quote",
-        ".${INFOMANIAK_REPLY_QUOTE_HTML_CLASS_NAME}",
-        ".moz-cite-prefix",
-        ".protonmail_quote",
-        ".yahoo_quoted",
-        ".zmail_extra", // Zoho
+        anyCssClassContaining("gmail_extra"),
+        anyCssClassContaining("gmail_quote"),
+        anyCssClassContaining(INFOMANIAK_REPLY_QUOTE_HTML_CLASS_NAME),
+        anyCssClassContaining("moz-cite-prefix"),
+        anyCssClassContaining("protonmail_quote"),
+        anyCssClassContaining("yahoo_quoted"),
+        anyCssClassContaining("zmail_extra"), // Zoho
         "[name=\"quote\"]", // GMX
         "blockquote[type=\"cite\"]",
```

Comment:
No idea who it can be this client, but if the standard is well respected then theoretically the algo on blockquotes should be enough, otherwise there will be a bug 🤔.
However, it can be deleted for the time being